### PR TITLE
Refactor FeebackResponsesLogic.java #6403

### LINKED ISSUE #6403: Refactor FeebackResponsesLogic.java
There are a few methods that are long and hard to read. Some refactorings can be made to improve readability.

### PR DESCRIPTION
**Pull request by ablyx**
Fixes #6403 

Refactor long method names for FeedbackResponsesLogic

### CODE REVIEW

File: src/main/java/teammates/logic/core/FeedbackResponsesLogic.java [hunk 1]
```diff
@@ -421,11 +426,32 @@ public void updateFeedbackResponse(
...
+            recreateResponse(newResponse, oldResponse);
+        }
+    }
+
+    /**
+     * Copy values that cannot be changed to defensively avoid invalid parameters.
+     * @param newResponse  values are copied from oldResponse
+     * @param oldResponse  values are copied to newResponse
+     */
+    private void copyFromOldToNew(FeedbackResponseAttributes newResponse,
```
Comment (chowyb) [inline src/main/java/teammates/logic/core/FeedbackResponsesLogic.java]: Still think the name of the method could be improved - perhaps `copyFixedValuesFromOldToNew`?

Comment (ablyx) [inline src/main/java/teammates/logic/core/FeedbackResponsesLogic.java]: Thanks for the tip, I couldn't think of a good name so I just wrote the javadocs. I'll make the necessary changes

File: src/main/java/teammates/logic/core/FeedbackResponsesLogic.java [hunk 2]
```diff
@@ -421,11 +426,32 @@ public void updateFeedbackResponse(
...
+                Assumption.fail();
+            }
+        } else {
+            // Recreate response to prevent possible future id conflict.
+            recreateResponse(newResponse, oldResponse);
+        }
+    }
+
+    /**
+     * Copy values that cannot be changed to defensively avoid invalid parameters.
```
Comment (damithc) [inline src/main/java/teammates/logic/core/FeedbackResponsesLogic.java]: `Copies`

Comment (ablyx) [inline src/main/java/teammates/logic/core/FeedbackResponsesLogic.java]: made the change!

**Pull request merged**